__tests__/__snapshots__/plugin.test.ts.snap

```diff
@@ -53,6 +71,21 @@ var f = function () {
 }();"
 `;
 
+exports[`babel plugin doesn't inject location for worklets in production builds 1`] = `
+"var _worklet_1852213185147_init_data = {
+  code: "function anonymous(){const x=1;}"
```

Comment:
I'm quite worried about the removal of `location` property, looks like we still use it here:

https://github.com/software-mansion/react-native-reanimated/blob/c6e7523541567a38dc4cb8d9c705b11e746a8cdf/src/reanimated2/initializers.ts#L47-L50

Maybe we should use `global.eval` when not in `__DEV__` mode?

Reply:
I agree with Tomek. 👍
Moreover, maybe we can use `location` and `sourceMap` only in debug mode. I could have a positive performance impact.

Reply:
We use it in debug only. `evalWithSourceMap` is injected only for dev builds.